src/Stryker.DataCollector/Stryker.DataCollector/CoverageCollector.cs

```diff
@@ -173,7 +173,7 @@ private void ReadConfiguration(string configuration)
             }
 
             var nameSpaceNode = node.SelectSingleNode("//Parameters/MutantControl");
-            if (nameSpaceNode != null && nameSpaceNode.Attributes != null)
+            if (nameSpaceNode?.Attributes != null)
```

Comment:
If you are changing this line, maybe use the "new' way of comparing

```suggestion
            if (nameSpaceNode?.Attributes is not null)
```

Reply:
Not possible: the DataCollector needs to be built against NetStandard 2.0, which forces C# to 7.3 whereas pattern matching features were introduced in C#9.

Reply:
Just add `<LangVersion>latest</LangVersion>` to the csproj?